apps/condo/domains/news/schema/GetNewsItemsRecipientsCountersService.test.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Generated by `createservice news.GetNewsItemsRecipientsCountersService --type mutations`
+ */
+const { map } = require('lodash')
+
+const {
+    makeLoggedInAdminClient,
+    makeClient,
+    expectToThrowAuthenticationError, expectToThrowAccessDeniedErrorToResult,
+} = require('@open-condo/keystone/test.utils')
+
+const { getNewsItemsRecipientsCountersByTestClient } = require('@condo/domains/news/utils/testSchema')
+const {
+    createTestOrganization,
+    createTestOrganizationEmployeeRole,
+} = require('@condo/domains/organization/utils/testSchema')
+const { createTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
+const { FLAT_UNIT_TYPE } = require('@condo/domains/property/constants/common')
+const { createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, createTestUser } = require('@condo/domains/user/utils/testSchema')
+
+const propertyMap = {
```

Comment:
You may check property test utils for this. It's already implemented functionality (property map creation)

Reply:
Yes, I've found some realizations but didn't find the one with `unitTypes`, so I decided to generate my own map as I need for testing.